debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/ByteCodeDumper.java

```diff
@@ -171,25 +171,30 @@ private static IDATA j9bcutil_dumpBytecodes(PrintStream out, J9ROMClassPointer r
 		UDATA start;
 		UDATA pc;
 		UDATA bc;
-		I32 low, high;
+		I32 low;
+		I32 high;
 		U32 npairs;
 		J9ROMMethodPointer romMethod = (J9ROMMethodPointer.cast(bytecodes.sub(J9ROMMethod.SIZEOF)));
 		U32 localsCount = new U32(ROMHelp.J9_ARG_COUNT_FROM_ROM_METHOD(romMethod).add(ROMHelp.J9_TEMP_COUNT_FROM_ROM_METHOD(romMethod)));
 		int resultArray[] = new int[8192];
-		String environment = "0";
-		boolean envVarDefined = false;
+		Integer dumpBytecodes = null;
 		int result;
 
-		if (System.getenv().containsKey("j9bcutil_dumpBytecodes")) {
-			envVarDefined = true;
-			environment = System.getenv().get("j9bcutil_dumpBytecodes");
+		try {
+			String environment = System.getenv().get("j9bcutil_dumpBytecodes");
+
+			if (environment != null) {
+				dumpBytecodes = Integer.valueOf(environment);
+			}
+		} catch (NumberFormatException e) {
```

Comment:
Not sure silent failure is the best behavior. An exception is more explicitly shows the problematic setting.

Reply:
No longer suppresses `NumberFormatException`.